applications/daos.md

```diff
@@ -0,0 +1,135 @@
+# W3F Grant Proposal
+***
+
+* **Project Name:** daos
+* **Team Name:** daos-org
+* **Payment Address:** 0x08A7876db92b07057E12fAF29AA3CCF33b5aAe3D (USDT ERC20)
+* **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
```

Comment:
Could you change this to level 1? Once this is done, I’m happy to mark it as ready for review and approve it.

Reply:
> Could you change this to level 1? Once this is done, I’m happy to mark it as ready for review and approve it.


thank you for your reply. I have made an update to Level.